listenbrainz/webserver/static/js/src/CBReviewModal.test.tsx

```diff
@@ -257,7 +251,7 @@ describe("submitReviewToCB", () => {
     const spy = jest.fn();
     instance.context.APIService.submitReviewToCB = spy;
 
-    await instance.submitReviewToCB(""); // access token not set
+    await instance.submitReviewToCB(); // access token not set
```

Comment:
How does this test work?
I can't see anywhere where you're un-setting the access token, and I assume the globalProps have a CB token set by default, right?

Reply:
I think this is currently the same test as the one below where `"does nothing if license was not accepted"`, looking at line 248

Reply:
Thanks for catching, I missed this while rewriting the submit function. Logic in tests should be fixed now :)